.github/workflows/README.md

```diff
@@ -18,6 +18,7 @@
    3. Deploy to multiple availability zones in the region - if checked, the cluster will be deployed to multiple availability zones in the region. Default value is `false`.
    4. Number of worker nodes to provision - number of compute nodes in the cluster. Default value is `2`.
 4. Wait for the workflow to finish.
+5. Optionally you can run the `task keepalive` from provision/openshift directory against your Openshift cluster to keep it alive.
```

Comment:
Should we also mention somewhere there is a periodic job destroying all clusters? Currently, it is not visible what is the purpose of `keepalive` namespace. Maybe we could create a new section called `Periodic jobs`?

Reply:
will add a new section for `Periodic Jobs`, thanks for the suggestion